.meta/transforms/dedupe.toml

```diff
@@ -0,0 +1,41 @@
+[transforms.dedupe]
+title = "Dedupe events"
+allow_you_to_description = "prevent duplicate Events from being outputted by using an LRU cache"
+beta = false
+common = false
+function_category = "filter"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/_component_options.toml", type: "transform", name: "dedupe") %>
+
+[transforms.dedupe.options.fields]
+type = "table"
+description = "Options controlling what fields to match against"
+required = true
+
+[transforms.dedupe.options.fields.children.match]
+type = "[string]"
+common = true
+examples = [["field1", "field2"]]
+default = ["timestamp"]
```

Comment:
Any reason this default is `timestamp` only? Wouldn't `["timestamp", "host", "message"]` be a better default. Note, this should use the [global `log_scheam` names](https://vector.dev/docs/reference/global-options/#log_schema).

Reply:
Done